src/main/java/org/sunbird/trending/controller/TrendingServiceImpl.java

```diff
@@ -178,84 +179,72 @@ private String getEndDateFormat() {
         return "";
     }
 
-    public Map<String, Object> trendingContentSearch(Map<String, Object> requestBody, String token) throws Exception {
+    public SBApiResponse trendingContentSearch(Map<String, Object> requestBody, String token) throws Exception {
+       SBApiResponse response = ProjectUtil.createDefaultResponse(Constants.MICROSITE_TOP_FEATURE_CONTENT_API);
+       try {
+           HashMap<String, Object> request = requestBody.containsKey(REQUEST) ? (HashMap<String, Object>) requestBody.get(REQUEST) : new HashMap<>();
+           if (request.isEmpty()) {
```

Comment:
Just used MapUtils and remove new HashMap<>

Reply:
Implimented as suggested.